### DISCUSSION TITLE
OSX: migrating Postgres db from one Mac to another

### DISCUSSION THREAD

**Discussion by carloposo-changan** [requarks/wiki#4129] [Help / Questions]
Hello,
my Mac unfortunately crashed. I have a Time Machine backup. 
Using the Migration Assistant on a new Mac allowed me to get all my documents back, but unfortunately Postgres is not able to start.

So I was trying to manually set-up my Postgres db.

Here are the steps:
- install a fresh copy of Postgres (v11)  
- replace the /Library/PostgreSQL/11/data folder with the one I dig out from my dead Mac, where I assume my data is stored
- check/fix owner of folders (must be 'postgres', I assume)
- start pgAdmin
- enter the PW for postgres user
- click on "PostgreSQL 11" server to show databases

This is where I have problems. I am prompted with the same pop-up asking me to enter the password for the user 'postgres'. I type it and I get the following:

![Screen Shot 2021-06-24 at 15 25 03](https://user-images.githubusercontent.com/84023086/123271004-af23b000-d500-11eb-9700-c1741d62aae6.png)

Any help will be very much appreciated.
Thanks.

**Comment by NGPixel** [comment]
I would suggest asking for help on stackoverflow or other postgres support channels as this is not really related to Wiki.js and more of a general postgres question.

**Reply by carloposo-changan** [reply]
Will do, thanks!
